encoders/numeric/UMAPEncoder/config.yml

```
@@ -0,0 +1,6 @@
+!UMAPEncoder
+with:
+  {}
```

Since the `model_path` is always required to have this encoder running properly, I'd suggest to remove the default value and leave a configuration in the YAML file.

```suggestion
  model_path: 'pre-trained.model'
```